plugins/BTagAnalyzer.cc

```diff
@@ -594,7 +606,6 @@ BTagAnalyzerT<IPTI,VTX>::BTagAnalyzerT(const edm::ParameterSet& iConfig):
   for(const auto& tag: iConfig.getParameter<std::vector<edm::InputTag> >("SubJets")){
     SubJetCollectionTags_.push_back(consumes<PatJetCollection>(tag));
   }
-  SubJetLabels_         = iConfig.getParameter<std::vector<std::string> >("SubJetLabels");
```

Comment:
I believe this line was removed by mistake.

Reply:
I removed it, as it is a duplicate of line 593 (four lines above).

Reply:
Ah, ok, did not notice that right away since I had to expand the ocde on GitHub to see that line. So this change is fine then. Thanks for spotting this.